.github/scripts/modify_dockerfile.sh

```diff
@@ -23,7 +23,7 @@ CONTENT_FOLDER="docker/"
 BINDER_DOCKERFILE="binder/Dockerfile"
 
 # Generate the sha1 for the content of the folder
-DOCKER_TAG="$(checksumdir -a sha1 $CONTENT_FOLDER)"
+DOCKER_TAG="$(checksumdir -i -e '__pycache__' -a sha1 $CONTENT_FOLDER)"
```

Comment:
Does this one give you the hash present in #231?

Reply:
```
$ checksumdir -a sha1 docker/
7fb962b09182c5f60ac2f80bbabb693290f3f741
```

```
checksumdir -i -e '__pycache__' -a sha1 docker/
7fb962b09182c5f60ac2f80bbabb693290f3f741
```
(no __pycache__ on my machine)

Reply:
I added the `__pycache__` as prevention. After I recreated the folder, I get this:

```
checksumdir -i -a sha1 docker                                                     
7fb962b09182c5f60ac2f80bbabb693290f3f741
```

```
checksumdir -a sha1 docker                                                            omaral-safi@Omars-MacBook-Pro
7fb962b09182c5f60ac2f80bbabb693290f3f741
```

Did you recreate the folder or did you just rebase?